### DISCUSSION TITLE
Cache problem after install

### DISCUSSION THREAD

**Discussion by dakisan** [bagisto/bagisto#6001] [Q&A]
After running the command line installation, and browsing to the homepage location I get:

`file_put_contents(D:\xampp-php7.3\htdocs\recyclo\my.site.com\storage\app/db-blade-compiler/views/3ad3cf66a3856b81120bd1be29475b50): failed to open stream: No such file or directory (View: D:\xampp-php7.3\htdocs\recyclo\my.site.com\packages\Webkul\Shop\src\Resources\views\home\index.blade.php)`

This is on Windows 10, xampp php 7.3 MariaDB 10.2. 

I have tried (without success)

`php artisan cache:clear`
`php artisan clear:config`
`php artisan config:cache`
`php artisan view:clear`

Hope you can help!

**Comment by rahulshukla-webkul** [comment]
Hi @dakisan
 
Please run the following commands
1. composer dump-autoload
2. php artisan vendor:publish

**Comment by andrew-za** [comment]
I had to chown all my files and directories from root to the correct user before my Bagisto installation worked.
`chown domain:psacln * -R`

**Comment by lahiruramesh** [comment]
still the same issue.

**Comment by prashant-webkul** [comment]
@LahiruR Please attach some screenshot the error or exception that you are getting?

**Comment by arieltoledo** [comment]
Same error till now using docker install too

**Comment by Sanjaybhattwebkul** [comment]
Hi @arieltoledo 
Can you please share the steps? followed by you.
And please attach a screen recording/screenshot for the same then we will understand better.

**Comment by arieltoledo** [comment]
@Sanjaybhattwebkul just following documentation on Docker install with composer. The failureless install I found was to follow the steps until the composer command  and then follow the web installer avoiding runing the next commands.

**Comment by muhammetfaik** [comment]
I deploy heroku my bagisto service but I have same error:

![Screenshot from 2023-06-21 23-24-24](https://github.com/bagisto/bagisto/assets/15978174/603a7e92-83b2-428d-8fb9-277e577dff08)

**Reply by devansh-webkul** [reply]
In the older version, we are using the db blade compiler. For proper functioning, you should have a db blade compiler folder in the storage directory, please check whether it is present or have proper permission.

**Reply by muhammetfaik** [reply]
how do I control  it is present or have proper permission and what is name db blade compiler folder ? Sorry I am beginner for bagisto and you think Bagisto install is simple heroku vs google cloud ?

**Reply by devansh-webkul** [reply]
In your screenshot, you can clearly check the folder name. In the recent version, this dependency is creating problems so we reduced this dependency and used the Laravel built-in facade.

Yes, installation is easy and simple. You can try Heroku as well. In the latest version, we removed triggers and functions as well because generally, this is also creating problems for many users.

**Reply by muhammetfaik** [reply]
![Screenshot from 2023-06-22 12-34-10](https://github.com/bagisto/bagisto/assets/15978174/bcdeeacc-3178-4969-8c35-e0d5e7cb923c)
dont show db blade compiler folder

**Reply by muhammetfaik** [reply]
what should I do?

**Reply by devansh-webkul** [reply]
Yes, this is the reason it is not working. Either installation is not done properly. Or someone deletes that folder. You can try re-creating the same name folder as views are stored here first.

Or you can try with `php artisan vendor:publish --all` also. If its new project and not done any changes you can force also `php artisan vendor:publish --all --force`
